recipe-server/normandy/recipes/tests/test_serializers.py

```diff
@@ -168,6 +168,42 @@ def test_validation_with_valid_data(self):
         }
         assert serializer.errors == {}
 
+    def test_update_action(self):
```

Comment:
I think the view tests are enough, this is straying a bit too far into testing DRF itself.